charts/promtail/Chart.yaml

```
@@ -3,7 +3,7 @@ name: promtail
 description: Promtail is an agent which ships the contents of local logs to a Loki instance
 type: application
 appVersion: 3.0.0
-version: 6.16.5
+version: 6.17.0
```

```suggestion
version: 6.16.6
```

I think patch version should be bumped.